src/rcc/enable.rs

```diff
@@ -0,0 +1,219 @@
+    CRC => (AHB1, crcen, crclpen, crcrst), // 12
+    DMA1 => (AHB1, dma1en, dma1lpen, dma1rst), // 21
+    DMA2 => (AHB1, dma2en, dma2lpen, dma2rst), // 22
+    OTG_HS_GLOBAL => (AHB1, otghsen, otghslpen, otghsrst), // 29
+
+    RNG => (AHB2, rngen, rnglpen, rngrst), // 6
+    OTG_FS_GLOBAL => (AHB2, otgfsen, otgfslpen, otgfsrst), // 7
+
+    FMC => (AHB3, fmcen, fmclpen, fmcrst), // 0
+    QUADSPI => (AHB3, qspien, qspilpen, qspirst), // 1
+
+    TIM2 => (APB1, tim2en, tim2lpen, tim2rst), // 0
+    TIM3 => (APB1, tim3en, tim3lpen, tim3rst), // 1
+    TIM4 => (APB1, tim4en, tim4lpen, tim4rst), // 2
+    TIM5 => (APB1, tim5en, tim5lpen, tim5rst), // 3
+    TIM6 => (APB1, tim6en, tim6lpen, tim6rst), // 4
+    TIM7 => (APB1, tim7en, tim7lpen, tim7rst), // 5
+    TIM12 => (APB1, tim12en, tim12lpen, tim12rst), // 6
+    TIM13 => (APB1, tim13en, tim13lpen, tim13rst), // 7
+    TIM14 => (APB1, tim14en, tim14lpen, tim14rst), // 8
+    LPTIM1 => (APB1, lptim1en, lptim1lpen, lptim1rst), // 9
+    WWDG => (APB1, wwdgen, wwdglpen, wwdgrst), // 11
+    SPI2 => (APB1, spi2en, spi2lpen, spi2rst), // 14
+    SPI3 => (APB1, spi3en, spi3lpen, spi3rst), // 15
+    USART2 => (APB1, usart2en, usart2lpen, usart2rst), // 17
+    USART3 => (APB1, usart3en, usart3lpen, usart3rst), // 18
+    UART4 => (APB1, uart4en, uart4lpen, uart4rst), // 19
+    UART5 => (APB1, uart5en, uart5lpen, uart5rst), // 20
+    I2C1 => (APB1, i2c1en, i2c1lpen, i2c1rst), // 21
+    I2C2 => (APB1, i2c2en, i2c2lpen, i2c2rst), // 22
+    I2C3 => (APB1, i2c3en, i2c3lpen, i2c3rst), // 23
+    CAN1 => (APB1, can1en, can1lpen, can1rst), // 25
+    PWR => (APB1, pwren, pwrlpen, pwrrst), // 28
+    DAC => (APB1, dacen, daclpen, dacrst), // 29
+    UART7 => (APB1, uart7en, uart7lpen, uart7rst), // 30
+    UART8 => (APB1, uart8en, uart8lpen, uart8rst), // 31
+
+    TIM1 => (APB2, tim1en, tim1lpen, tim1rst), // 0
+    TIM8 => (APB2, tim8en, tim8lpen, tim8rst), // 1
+    USART1 => (APB2, usart1en, usart1lpen, usart1rst), // 4
+    USART6 => (APB2, usart6en, usart6lpen, usart6rst), // 5
+    ADC1 => (APB2, adc1en, adc1lpen, adcrst), // 8
+    ADC2 => (APB2, adc2en, adc2lpen, adcrst), // 9
+    ADC3 => (APB2, adc3en, adc3lpen, adcrst), // 10
+    SDMMC1 => (APB2, sdmmc1en, sdmmc1lpen, sdmmc1rst), // 11
+    SPI1 => (APB2, spi1en, spi1lpen, spi1rst), // 12
+    SPI4 => (APB2, spi4en, spi4lpen, spi4rst), // 13
+    SYSCFG => (APB2, syscfgen, syscfglpen, syscfgrst), // 14
+    TIM9 => (APB2, tim9en, tim9lpen, tim9rst), // 16
+    TIM10 => (APB2, tim10en, tim10lpen, tim10rst), // 17
+    TIM11 => (APB2, tim11en, tim11lpen, tim11rst), // 18
+    SPI5 => (APB2, spi5en, spi5lpen, spi5rst), // 20
+    SAI1 => (APB2, sai1en, sai1lpen, sai1rst), // 22
+    SAI2 => (APB2, sai2en, sai2lpen, sai2rst), // 23
+}
+
+#[cfg(any(feature = "svd-f730", feature = "svd-f7x2", feature = "svd-f7x3",))]
+bus! {
+    AES => (AHB2, aesen, aeslpen, aesrst), // 4
+
+    SDMMC2 => (APB2, sdmmc2en, sdmmc2lpen, sdmmc2rst), // 7
```

Comment:
The `SDMMC2` is not only available in `svdf7x2` and `svd-f7x3`:

``` bash
stm32f7xx/svd$ rg SDMMC2 -l --sort-files
STM32F7x2.svd
STM32F7x3.svd
STM32F7x7.svd
STM32F7x9.svd
```

Reply:
`f765`, `f7x7` and `f7x9` doesn't have `sdmmc2en` and `sdmmc2rst` fields in `RCC`.
This should be fixed in `stm32-rs` first